src/components/Search/Search.css

```diff
@@ -0,0 +1,30 @@
+.search__flex {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+input {
+  width: 900px;
+  height: 41px;
```

Comment:
Usually you're fucked if your input is constant width, try changing screen size.

Height is more or less ok. It usually lives in a variable somewhere to make sure all inputs in the app are the same height.